principles-of-providing-excellent-support/frictionless-support.md

```diff
@@ -0,0 +1,85 @@
+
+### Mechanics of Escalation
+
+The point of escalating a ticket is to signal to a Senior Support Technician "I don't know what else to do to resolve this issue." For mechanics of _how_ to do certain tasks, [Slack calls](../communicating-as-a-team/using-slack.md) are a better choice. 
+
+If a Support Technician finds themselves stuck on what else to do, we escalate the ticket via a note in Help Scout. The singular focus of that note is to bring the Senior Support Technician up to speed on the ticket without having to read through the entire thing. We use the canned response in Help Scout to answer most thoroughly. 
+
+Once the ticket has been escalated, **we leave it assigned to the original technician** unless otherwise instructed. This allows for the technician to monitor whether the Senior Support Technician has gotten to it, and if they haven't, to either remind them or notify the customer before logging off for the day. 
+
+### Angry Customers
+The other reason for escalating a ticket is if the customer is demonstrably angry or belligerent. Often an angry customer (in addition to simply needing to be handled per our [tone guide](tone-guide.md)'s principle of "results oriented") simply needs to hear from a different technician, and they respond well to titles like "Senior," "Director," or "Head." 
+
+In the case of an angry customer, we have a conversation in Slack about it, deciding whether escalation is necessary. 
+
+## Assigning Tickets To Other Technicians
+
+The Senior Support technician has the role of "Queue Master" which includes assigning tickets to specific support technicians to ensure that everyone has a fair load of responsibilities. 
+
+It's also appropriate in the interest of frictionless support for technicians to communicate among themselves and assign tickets as appropriate. 
+
+Valid reasons for assigning a ticket to another technician (with their knowledge and permission) include:
+- the technician has more experience with a certain payment gateway, add-on, or functionality. 
+- the technician has a local install better suited for testing and reproducing the customer's issue.
+- the technician is in a time zone better suited for helping the customer in need of more "real-time" assistance.
+
+The key to assigning things to other technicians is that it requires a conversation via Slack (in the public support channel) where both technicians agree to it. 
+
+## Involving other Departments
+
+Often in the course of responding to a technical issue, the customer will have a question about their account at GiveWP, or requesting a Demo from the Customer Success team, or managing licenses (for example). 
+
+In order to keep things frictionless, the best thing we as Support Technicians can do is to loop in the appropriate department ourselves, and handle as much as possible for the customer. 
```

Comment:
Change to
"In order to keep the customer experience frictionless, Support Technicians should reassign tickets to the appropriate person or department immediately with a note that helps focus what action should be taken. This way the next interaction the customer receives is actionable, and not a reassignment update."